### PR TITLE
Feat offline events

### PR DESCRIPTION
**Pull request by b-ma**
Implements `statechange` and `complete` events for `OfflineAudioContext`

### CODE REVIEW

File: Cargo.toml
Comment (b-ma) [inline Cargo.toml:17]: Need to wait for upstream release before merging

**Review by orottier — APPROVED** [review]

File: Cargo.toml
Comment (orottier) [reply Cargo.toml:17]: Yup, will release later today

**Pull request merged**
